docs/UserGuide.md

```diff
@@ -288,60 +261,113 @@ Examples:
+* Appointment Date: Valid dates (after current date), in the format of `yyyy-mm-dd`.
+
+#### Name
+
+![edit format](images/find_n0.png)
+
+#### Precise expected outputs on success:
+* Successful find message. ‘X persons listed!’, where X is the number of contacts listed.
   The list of contacts whose names contain the input name is shown.
 
-![edit format](images/find1.png)
+![edit format](images/find_n1.png)
 
-* Precise expected outputs on failure:
-* If no name is input after the find command, an error message explaining the error will be shown,
-  reminding the user to follow the correct input format.
+##### Precise expected outputs on failure:
+If no name is input after the find command, an error message explaining the error will be shown, 
+reminding the user to follow the correct input format.
 
-![edit format](images/find2.png)
+* Error Message:
 
-### Locating persons by address: `find_add`
+```
+Invalid command format! find n/: Finds all persons whose names contain any of the specified keywords (case-insensitive) and displays them as a list with index numbers.
+```
 
-Finds persons whose address contain any of the given keywords.
+![edit format](images/find_n2.png)
 
-Format: `find_add KEYWORD [MORE_KEYWORDS]`
+#### Address
 
-* The search is case-insensitive. e.g `tokyo` will match `Tokyo`.
-* The order of the keywords does not matter. e.g. `Little Tokyo` will match `Tokyo Little`.
-* Only the address is searched.
-* Only full words will be matched e.g. `Toky` will not match `Tokyo`.
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Little Geylang` will return `Little Tokyo`, `Tokyo`.
+![edit format](images/find_a0.png)
 
-Examples:
-* `find_add geylang` returns all users whose addresses contain `geylang`.
+#### Precise expected outputs on success:
+* Successful find message. ‘X persons listed!’, where X is the number of contacts listed.
+  The list of contacts whose address contain the input address is shown.
 
-![edit format](images/findadd0.png)
+![edit format](images/find_a1.png)
 
-* Precise expected outputs on success:
-* Successful find message. ‘X persons listed!’’, where X is the number of contacts listed.
-  The list of contacts whose address contains the input address is shown.
+##### Precise expected outputs on failure:
+If no address is input after the find command, an error message explaining the error will be shown,
+reminding the user to follow the correct input format.
 
-![edit format](images/findadd1.png)
+* Error Message:
 
-* Precise expected outputs on failure:
-* If no name is input after the find command, an error message explaining the error will be shown,
-  reminding the user to follow the correct input format.
+```
+Invalid command format! find a/: Finds all persons whose address contain any of the specified keywords (case-insensitive) and displays them as a list with index numbers.
+```
+
+![edit format](images/find_a2.png)
+
+#### Appointment Date
```

Comment:
I think instead of making it have the same heading as the rest, like Format, Example commands, and so on, which make it harder to differentiate, maybe can italicize them

Reply:
Fixed!